cupy/core/_kernel.pyx

```diff
@@ -981,8 +986,8 @@ cdef class ufunc:
         return kern
 
 
-cpdef create_ufunc(name, ops, routine=None, preamble='', doc='',
-                   default_casting=None, loop_prep=''):
+cdef list _get_ops(ops, routine):
+    """Parse dtype character mapping"""
```

Comment:
It's difficult to tell what's the difference between inputs and outputs. Can you improve it?

Reply:
I think I'm going to address this later.